### PR TITLE
Added reference to XenoTactic youtube channel & add documentation on alignment/centering.

### PR DESCRIPTION
**Pull request by Kietyo**
I formatted the file using Intellij's "Reformat File" which might explain some of the formatting changes.

### CODE REVIEW

File: korge/reference/views_standard/index.md [hunk 1]
```diff
@@ -1,9 +1,5 @@
 ---
-layout: default
-title: Standard Views
-title_prefix: KorGE Views
-fa-icon: fa-object-ungroup
-priority: 6
+layout: default title: Standard Views title_prefix: KorGE Views fa-icon: fa-object-ungroup priority: 6
```
Comment (soywiz) [inline korge/reference/views_standard/index.md]: This seems wrong, right? Was this an issue related to line endings?

File: korge/reference/views_standard/index.md [hunk 2]
```diff
@@ -290,37 +329,71 @@ class ScaleView(width: Int, height: Int, scale: Double = 2.0, var filtering: Boo
...
+}
+interface IHtml {
+    var html: String
+}
 
 fun View?.setText(text: String) = run { this.foreachDescendant { if (it is IText) it.text = text } }
 fun View?.setHtml(html: String) = run { this.foreachDescendant { if (it is IHtml) it.html = html } }
 ```
+
+## Alignment / Centering
```
Comment (soywiz) [inline korge/reference/views_standard/index.md:375]: Nice!

File: korge/samples/index.md [hunk 3]
```diff
@@ -27,7 +23,12 @@ status: updated
 * [KorGE Tutorial #6 - Resolution handling and coordinate system](https://www.youtube.com/watch?v=81IG0ld5w-8)
 * [KorGE Tutorial #7 - Sprites and SpriteAnimations from XML or JSON files](https://www.youtube.com/watch?v=atElzA2jYkQ)
 
-Game tutorials:
+### XenoTactic Korge Tips
+
+XenoTactic is a community member who frequently creates videos on Korge. Check his channel
```
Comment (soywiz) [inline korge/samples/index.md:32]: Nice! Maybe you can also put links to individual videos so they appear in the search and they are easier to find?

**Review by soywiz — COMMENTED** [review]
It seems that `korge/reference/views_standard/index.md` yaml is wrong, can you fix it?

File: korge/reference/views_standard/index.md (hunk 1 above)
Comment (Kietyo) [reply korge/reference/views_standard/index.md]: This was from auto-format. Reverted.

**Comment by Kietyo** [comment]
Done

File: korge/samples/index.md [hunk 4]
```diff
@@ -1,24 +1,20 @@
 ---
-layout: default
-title: "Samples"
-title_prefix: KorGE
-fa-icon: fa-chalkboard-teacher
-priority: 2
-status: updated
+layout: default title: "Samples"
```
Comment (soywiz) [inline korge/samples/index.md]: Layout and title seems still to be on the same line. Can you fix this?

File: korge/samples/index.md [hunk 5]
```diff
@@ -1,24 +1,20 @@
...
+title_prefix: KorGE fa-icon: fa-chalkboard-teacher priority: 2 status: updated
```
Comment (soywiz) [inline korge/samples/index.md]: This line also have several keys in the same line. Can you fix them?

**Comment by Kietyo** [comment]
done

**Comment by soywiz** [comment]
Thanks!

63mraw

**Pull request merged**